test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperEditorExtensionsTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.TestCommon;
+using Microsoft.AspNetCore.Mvc.ViewEngines;
+using Moq;
+using Xunit;
+using static Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperDisplayExtensionsTest;
```

Comment:
Get rid of this `using static`. Shouldn't be necessary for anything and it's much easier to read the code if it's explicit e.g. change the parameter types to `HtmlHelperDisplayExtensionsTest.FormatModel`.